gitCore/jGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreRepository.java

```diff
@@ -134,7 +134,32 @@ private <T> T withRevWalkUnchecked(CheckedFunction1<RevWalk, T> fun) {
   }
 
   private boolean isBranchPresent(String branchFullName) {
-    return Try.of(() -> jgitRepoForMainGitDir.resolve(branchFullName)).getOrNull() != null;
+    // If '/' chars exist in the branch name, then this loop-based testing is needed in order to avoid possible IDE errors,
```

Comment:
I would rather move this comment onto the method. That will allow us to make it a javadoc and actually reference classes and methods (and gives more flexibility in the scope of formatting IIRC).

Reply:
Although... this entire comment isn't relevant in terms of method interface, it's an implementation detail 🤔

Reply:
I would rather...  for the sake of the other methods/classes refs and better formatting (HTML). but no strong feelings about it